test/unit/com/unifina/service/EthereumIntegrationKeyServiceSpec.groovy

```diff
@@ -267,6 +268,16 @@ class EthereumIntegrationKeyServiceSpec extends Specification {
 		SecUser.count == 1
 	}
 
+	void "createEthereumUser() creates inbox stream"() {
```

Comment:
Add tests:
- Adding Ethereum ID should create inbox stream
- Adding Ethereum private key (`IntegrationKey` of type `Ethereum`) should create inbox stream